lisa/tools/echo.py

```diff
@@ -18,6 +18,24 @@ def _windows_tool(cls) -> Optional[Type[Tool]]:
     def _check_exists(self) -> bool:
         return True
 
+    def write_to_file(
+        self,
+        val: str,
+        file_path: str,
+        force_run: bool = False,
```

Comment:
The force_run doesn't need to be a parameter, and it can be True always in this scenario. We want every time the write operation is executed.